Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_AWS_Enrichment.yml

```
@@ -1576,8 +1576,51 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 440,
-          "y": 3130
+          "x": 450,
+          "y": 3220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: 85485dba-f2c7-42c2-864e-b695a9ee0551
+    type: regular
+    task:
+      id: 85485dba-f2c7-42c2-864e-b695a9ee0551
+      version: -1
+      name: Set system IDs grid field (type)
+      description: Sets the Azure Compute instance ID to the grid field for the ASM system IDs object.
```

```suggestion
      description: Sets the Compute instance ID to the grid field for the ASM system IDs object.
```
remove Azure reference

---

resolved in https://github.com/demisto/content/pull/29098/commits/7faeccb53ba3109b146c85d3643ad539a268dcd0